.babelrc

```diff
@@ -12,6 +12,12 @@
     "@babel/plugin-transform-flow-strip-types",
     "@babel/plugin-transform-property-mutators",
     "@babel/plugin-proposal-object-rest-spread",
+    [
+      "@babel/plugin-transform-spread",
+      {
+        "loose": true
```

Comment:
loose is required?

Reply:
yes

Reply:
@dan-ziv can you please add explanation why

Reply:
The error is raised from `toChildArray` API of preact.
From this method:
`return EMPTY_ARR.concat.apply([], children.map(toChildArray));`
The plugin doesn't know how to spread empty arrays without loose transpiling.

Reply:
@Yuvalke please see above